one failed test 

```

FAILED TESTS:
  HeartBeat
    beat method
      ✖ should call setInactive
        HeadlessChrome 67.0.3396 (Linux 0.0.0)
      AssertError: expected setInactive to be called once but was called 0 times
          at Object.fail (kolibri/core/assets/test/heartbeat.js:4392:21)
          at failAssertion (kolibri/core/assets/test/heartbeat.js:4351:16)
          at Object.assert.(anonymous function) [as calledOnce] (kolibri/core/assets/test/heartbeat.js:4376:13)
          at kolibri/core/assets/test/heartbeat.js:50400:62
      ✖ should call wait
        HeadlessChrome 67.0.3396 (Linux 0.0.0)
      AssertError: expected wait to be called once but was called 0 times
          at Object.fail (kolibri/core/assets/test/heartbeat.js:4392:21)
          at failAssertion (kolibri/core/assets/test/heartbeat.js:4351:16)
          at Object.assert.(anonymous function) [as calledOnce] (kolibri/core/assets/test/heartbeat.js:4376:13)
          at kolibri/core/assets/test/heartbeat.js:50406:62
      ✖ should set timerId to a setTimeout identifier
        HeadlessChrome 67.0.3396 (Linux 0.0.0)
      AssertionError: expected 'undefined' to equal 'number'
      + expected - actual
      -undefined
      +number
```

--- a/kolibri/core/assets/test/heartbeat.js
+++ b/kolibri/core/assets/test/heartbeat.js
@@ -57,6 +57,7 @@ describe('HeartBeat', function() {
     beforeEach(function() {
       this.heartBeat = new HeartBeat();
       this.heartBeat.active = false;
+      this.heartBeat.enabled = true;
       this.store = coreStore.factory();
       this.checkSessionStub = sinon.stub(this.heartBeat, 'checkSession');
       this.checkSessionStub.resolves();

